### ISSUE TITLE
5.01 x64 Executable Not Working

### ISSUE THREAD

**Issue by parasupaman** [rigaya/QSVEnc#55]
Anyone else having issues with builds after QSVEncC64 4.13 on Tiger Lake CPUs?

Seems unable to start encoding, when running any QSVEnc version after 4.13 on Windows 10 - Version 20H2 (OS Build 19042.985).  Notice that the executable simply stops and exits without returning any kind of error.  Even using  --log-level debug  generates nothing.

```
D:\MKV\HDR>QSVEncC64.exe --codec hevc -i Transcendence.h264 -o Test.hevc
--------------------------------------------------------------------------------
Test.hevc
--------------------------------------------------------------------------------

D:\MKV\HDR>
```

Version 4.13 works fine:

```
D:\MKV\HDR>QSVEncC64.exe --codec hevc -i Transcendence.h264 -o Test.hevc
--------------------------------------------------------------------------------
Test.hevc
--------------------------------------------------------------------------------
cop3.DirectBiasAdjustment value changed off -> auto by driver
cop3.GlobalMotionBiasAdjustment value changed off -> auto by driver
QSVEncC (x64) 4.13 (r2009) by rigaya, Feb 17 2021 13:49:19 (VC 1928/Win/avx2)
OS             Windows 10 x64 (19042) [UTF-8]
CPU Info       11th Gen Intel Core i7-1165G7 @ 2.80GHz [TB: 4.69GHz] (4C/8T) <Tigerlake>
GPU Info       Intel Iris(R) Xe Graphics (96EU) 100-1300MHz [28W] (27.20.100.9466)
Media SDK      QuickSyncVideo (hardware encoder) PG, 1st GPU, API v1.34
Async Depth    5 frames
Buffer Memory  d3d9, 1 input buffer, 21 work buffer
Input Info     avqsv: H.264/AVC, 1920x1080, 25/1 fps
AVSync         cfr
Output         HEVC main @ Level 4 (high tier)
               1920x1080p 1:1 25.000fps (25/1fps)
Target usage   4 - balanced
Encode Mode    Constant QP (CQP)
CQP Value      I:24  P:26  B:27
QP Limit       min: 1, max: 51
Trellis        Auto
Ref frames     3 frames
Bframes        3 frames, B-pyramid: on
Max GOP Length 250 frames
Ext. Features  WeightP WeightB QPOffset tskip ctu:64 sao:all
7...
```

However, checking features using version 5.01 appears to work and detects my CPU correctly as below:

```
D:\MKV\HDR>QSVEncC64.exe --check-features
Checking for QSV...
QSVEncC (x64) 5.01 (r2280) by rigaya, May  8 2021 12:05:29 (VC 1928/Win/avx2)
 reader: raw, avi, avs, vpy, avqsv [H.264/AVC, H.265/HEVC, MPEG2, VP8, VP9, AV1]
Environment Info
OS : Windows 10 x64 (19042) [UTF-8]
CPU: 11th Gen Intel Core i7-1165G7 @ 2.80GHz [TB: 4.71GHz] (4C/8T) <Tigerlake>
RAM: Used 8479 MB, Total 16009 MB
GPU: Intel Iris(R) Xe Graphics (96EU) 100-1300MHz [28W] (27.20.100.9466)

Media SDK Version: Hardware API v1.34

Supported Enc features:
Codec: H.264/AVC
             CBR   VBR   AVBR  QVBR  CQP   LA    LAHRD ICQ   LAICQ VCM
RC mode       o     o     o     o     o     o     o     o     o     o
10bit depth   x     x     x     x     x     x     x     x     x     x
Fixed Func    o     o     o     o     o     x     x     o     x     o
...
```

**Comment by parasupaman** [comment]
SOLUTION:

Finally figured it out.  It works if I specify the OUTPUT FILE as MKV (eg. "-o Test.mkv").  Specifying a raw format (eg. "-o Test.hevc"), no longer works as in previous versions (<= 4.13).

**Comment by rigaya** [comment]
Thank you for pointing out the issue, I'll have this fixed in the next release.

**Comment by parasupaman** [comment]
**FIXED in version 5.03**.  Closing issue.
